.github/workflows/tests-e2e-upgrade.yaml

```diff
@@ -130,7 +132,7 @@ jobs:
       job_name: 'Setup & Test'
     strategy:
       fail-fast: false
-      max-parallel: 32
+      max-parallel: 40
```

Comment:
Sometimes I wonder if we shouldn't just set this to 100 and stop incrementing it PR by PR :sweat_smile:

Reply:
ack, that's definitely a papercut now - modifying this value when the total set of configs is built from multiple files.